src/commands/install.yml

```diff
@@ -1,18 +1,23 @@
 description: |
-  Install specified version of terraform.
+  Install specified version of terraform. If unspecified, the latest version will be installed.
 
 parameters:
   terraform_version:
     type: "string"
     description: "Specify version of terraform to install."
-    default: "0.13.5"
+    default: "latest"
+  platform:
+    type: enum
+    description: |
+      Specify the target platform to install Terraform to. By default, linux_amd64 will be used.
+      Options: [linux_amd64, linux_arm]
+    enum: [linux_amd64, linux_arm]
```

Comment:
We should go ahead and add darwin to this list for macOS executors (if... someone wanted to use it?). I would suggest Windows as well, but might be tricky given shell issues.